internal/cleanup/config.go

```diff
@@ -28,10 +29,11 @@ var _ setup.DBConfigProvider = (*Config)(nil)
 // Config represents the configuration and associated environment variables for
 // the cleanup components.
 type Config struct {
-	Port     string        `envconfig:"PORT" default:"8080"`
-	Timeout  time.Duration `envconfig:"CLEANUP_TIMEOUT" default:"10m"`
-	TTL      time.Duration `envconfig:"CLEANUP_TTL" default:"336h"`
-	Database *database.Config
+	Port          string        `envconfig:"PORT" default:"8080"`
+	Timeout       time.Duration `envconfig:"CLEANUP_TIMEOUT" default:"10m"`
+	TTL           time.Duration `envconfig:"CLEANUP_TTL" default:"336h"`
+	Database      *database.Config
+	BlobstoreType string `envconfig:"BLOBSTORE_TYPE" default:"GCS"`
```

Comment:
Throughout - Change GCS to GoogleCloudStorage

assume the TLA isn't well known outside of Google.